### DISCUSSION TITLE
More choices for Date selector in entries listing

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4650] [Feature Requests]
### Suggest an idea for ExpressionEngine

![image](https://github.com/user-attachments/assets/c4141679-4a5b-40cd-8f66-6555d5bed4fa)

It would be great to have additional options here:

- Custom date range
- This week
- Last week
- This month
- Last month
- This year
- Last year

**Comment by intoeetive** [comment]
Date range might be a good addition

@Yulyaswan is there an easy way for us to add a picker for date range?
